README.md

```diff
@@ -110,9 +111,17 @@ And, more specifically than that, image-based profiling readouts from [CellProfi
 
 Therefore, we have included some custom tools in `pycytominer/cyto_utils` that provides other functionality:
 
-- [CellProfiler CSV collation](#CellProfiler-CSV-collation)
-- [Cell locations lookup table generation](#Creating-a-cell-locations-lookup-table)
-- [Generating gct files for Morpheus visualization](#Generating-a-GCT-file-for-morpheus)
+- [Data processing for image-based profiling](#data-processing-for-image-based-profiling)
```

Comment:
Why did this block update?

Reply:
The Markdown linter pre-commit might have detected the usage of a table of contents in the README and found headers that were not initially included in the table of contents. It probably automatically updated the table of contents to accurately reflects the document's header structure.